### ISSUE TITLE
Fails to compile (ramda) - 0.2.0

### ISSUE THREAD

**Issue by schuchard** [briebug/ngrx-auto-entity#50]
- yarn add @briebug/ngrx-auto-entity@~0.2
- add to root module
```ts
	...
    NgrxAutoEntityModule.forRoot()
  ],
  bootstrap: [AppComponent],
})
export class AppModule {}
```

```bash
➜ yarn start
yarn run v1.16.0
$ ng serve

                                                                                                                                                 
ERROR in ./node_modules/@briebug/ngrx-auto-entity/fesm2015/briebug-ngrx-auto-entity.js
Module not found: Error: Can't resolve 'ramda' in '/Users/schuchard/dev/repos/open-bike/node_modules/@briebug/ngrx-auto-entity/fesm2015'
** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
ℹ ｢wdm｣: Failed to compile.
```

```
Node: v10.15.1
"@angular/core": "~8.1.0",
```

**Comment by jrista** [comment]
Thanks for the report, @schuchard. Looks like there was a straggler reference to ramda, even though I'd removed it from the library as a peer dependency.

**Comment by jrista** [comment]
Fixed by #51
